tensorrec/eval.py

```diff
@@ -59,6 +59,77 @@ def recall_at_k(model, test_interactions, k=10, user_features=None, item_feature
     return hit / retrieved
 
 
+def _idcg(hits, k=10, ctype="binary"):
```

Comment:
We could move this declaration inside of the ndcg() function to avoid crowding the module-level namespace.

Reply:
Ahh nevermind -- its nice to be able to test this independently, like you have 👍